### PR TITLE
<CustomModalLayout/> - add dropdown example for storybook

### PR DESCRIPTION
**Pull request by sivanharel**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6026

File: src/CustomModalLayout/docs/examples/DropdownExample.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+/* eslint-disable */
+import React from 'react';
+import { Checkbox, Dropdown } from 'wix-style-react';
```
Comment (TalZa1810) [inline src/CustomModalLayout/docs/examples/DropdownExample.js:3]: no need for the imports. it renders this without the imports.

**Review by TalZa1810 — APPROVED** [review]
looks good, just a tiny note.

File: src/CustomModalLayout/docs/examples/DropdownExample.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      <CustomModalLayout
+        primaryButtonText="Save"
+        secondaryButtonText="Cancel"
+        onCloseButtonClick={() => {}}
+        title="Create New Coupon"
+        subtitle="Make customers come back to your store with coupons"
+        sideActions={<Checkbox>Checkbox</Checkbox>}
+      >
+        <Dropdown
+          popoverProps={{ appendTo: 'window' }}
```
Comment (EgoziE) [inline src/CustomModalLayout/docs/examples/DropdownExample.js:15]:
```suggestion
          popoverProps={{ appandTo: 'window' }}
```

**Review by EgoziE — CHANGES_REQUESTED** [review]

File: src/CustomModalLayout/docs/examples/DropdownExample.js (hunk 2 above)
Comment (EgoziE) [reply src/CustomModalLayout/docs/examples/DropdownExample.js:15]: #sorry

**Review by milkyfruit — CHANGES_REQUESTED** [review]
@sivanharel please add the descriptions to the example:

**Opening popovers form a modal**
CustomModalLayout doesn't let to overflow the content. To make popover elements work properly they need to be used with popover prop `appendTo` set to `window`.

**Pull request merged**